cincinnati/src/plugins/internal/cincinnati_graph_fetch.rs

```diff
@@ -263,6 +278,7 @@ mod tests {
     fetch_upstream_failure_test!(
         name: fetch_fail_invalid_url,
         upstream: "invalid.url",
+        gb_timeout: 30,
```

Comment:
looks like all occurrences of the `fetch_upstream_failure_test!` macro use the same timeout value. According to the DRY principle it can therefore be hardcoded in the macro.

Reply:
Randomized values for `gb_timeout` there. These are not being verified yet, but it may trigger an error so its better than hardcoding.

Does that sound acceptable?

Reply:
I don't think so as I find it too misleading.

Reply:
Hardcoded this in the macros so that won't lead us into thinking this is actually properly tested

Reply:
I think we still have a misunderstanding happening here. What I mean by hardcoding in the macro (note the singular) is to use a fixed number in the macro itself, instead of exposing a macro argument for it. From me macro's point of view this is still dynamically passed in.